### PR TITLE
The author information of the testimonial section, spreads out from the author div on mobile devices

### LINKED ISSUE #1283: The author information of the testimonial section, spreads out from the author div on mobile devices.
**Describe the bug**
The author information of the testimonial section, spreads out from the author div on mobile devices.  Furthermore some margin and padding values didn't set properly on mobile devices.

**To Reproduce**
1. Go to https://sefglobal.org/
2. Click on inspect elements
3. Scroll down to testimonial section

**Expected behavior**
set particular margin, padding values and set author div's height as auto to make sure, content is not spreading out of the div.

**Screenshots**
githubTestimonialSection

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [google chrome]
 - Version [106.0.5249.103]

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by sathmikaDev**
## Purpose
The purpose of this PR is to fix #1283 

## Goals
make author div in testimonial section responsive to make sure author's details are not spreading out of the div

## Approach
change author div's height as `auto` to ensure all the details fit inside the div. make author div's `grid-template-rows: auto auto;` and `grid-template-columns: auto auto;` 
did some changes on margin, padding values.

### Screenshots
iphonese

  
### Preview Link
https://pr-1288-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: assets/css/global-custom.css [hunk 1]
```diff
@@ -43,11 +43,13 @@ body {
...
     margin-top: -25px;
     padding-left: 5px;
-    height: 60px;
+    /* set height auto */
+    height: auto;
     display: inline-grid;
     grid-template-columns: 55px auto;
     grid-template-rows: auto 25px;
-    padding: 5px 30px 5px 5px;
+    /* reduced padding right to 15px */
```
Comment (anjula-sack) [inline assets/css/global-custom.css]: can we remove the comments?

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Good job! @Sathmikasenadheera01 Let's remove the comments

File: assets/css/global-custom.css (hunk 1 above)
Comment (sathmikaDev) [reply assets/css/global-custom.css]: yeah sure.

**Review by anjula-sack — APPROVED** [review]
Thank you! @Sathmikasenadheera01

**Pull request merged**